pkg/registry/core/pod/rest/subresources.go

```diff
@@ -113,7 +115,24 @@ func (r *AttachREST) Connect(ctx context.Context, name string, opts runtime.Obje
 	if err != nil {
 		return nil, err
 	}
-	return newThrottledUpgradeAwareProxyHandler(location, transport, false, true, responder), nil
```

Comment:
rather than copying the construction of the upgrade aware proxy, shouldn't we still call `newThrottledUpgradeAwareProxyHandler` and wrap the result if the feature is enabled?

```go
handler := newThrottledUpgradeAwareProxyHandler(location, transport, false, true, responder)
if utilfeature.DefaultFeatureGate.Enabled(features.ClientRemoteCommandWebsockets) {
  ... construct translator ...
  handler = proxy.NewTranslatingHandler(handler, translator)
}
return handler, nil
```

Reply:
Fixed.